po/es.po

```diff
@@ -11128,10 +11129,11 @@ msgid ""
 "contrast\n"
 "'averaged guided filter' is a geometric mean of both methods"
 msgstr ""
-"'no' afecta el contraste global y local (seguro si solo agrega contraste)\n"
-"'filtro guiado' solo afecta el contraste global tratando de preservar el "
+"No: afecta el contraste global y local (es seguro si solo se añade "
+"contraste)\n"
+"Filtro guiado: solo afecta el contraste global tratando de preservar el "
```

Comment:
Options were quoted in original sentences, why the change to colon as it was a definition?
I think, that it will remain as quoted word followed with an small explanation.

Reply:
The score changes according to the language.